apps/prairielearn/src/batched-migrations/20250214171608_course_instance_usages__external_grading_jobs__backfill.sql

```diff
@@ -57,6 +57,7 @@ FROM
 WHERE
   gj.grading_method = 'External'
   AND gj.grading_finished_at IS NOT NULL
+  AND gj.grading_received_at IS NOT NULL
```

Comment:
Stupid nit, ordering them by expected sequence:

```suggestion
  AND gj.grading_received_at IS NOT NULL
  AND gj.grading_finished_at IS NOT NULL
```